config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+
+  post "/guests/:id", to: "guest_rooms#create"
```

Comment:
A more RESTful route here would be "/guests/:id/rooms" just to indicate that it's posting to a JOINS record rather than creating a new guest (which is what the current path suggests)